### DISCUSSION TITLE
How to check friends current status ?

### DISCUSSION THREAD

**Discussion by kingprapop** [Terrance/SkPy#201] [Q&A] [answered]
I can't found way to check that.
If anyone know to do? , please share your experience.

Normaly ,I can get SkypePresenceEvent when friends change thier status ,But friends continue "Online,Away" i can't not know them status

Target:  I want to sendmsg to only who Online(Active).

**Comment by Terrance** [comment, marked as answer]
You'll have to keep track of presence events yourself (by recording the last-seen `SkypePresenceEvent` for each contact), as Skype doesn't provide a direct API for reading a user's status.

If you have contacts that are persistently idle, they're probably mobile clients that advertise themselves as always-available (receiving push notifications even if the app isn't open), so you'll probably want to require an online status.
